h5p-bildetema/src/components/Toggle/ReactToggle.scss

```
@@ -0,0 +1,117 @@
+@use "../../../../common/abstracts" as *;
+
+.react-toggle {
+  touch-action: pan-x;
+
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+  background-color: transparent;
+  border: 0;
+  padding: 0;
+  user-select: none;
```

I think we could keep the `-webkit-touch-callout: none;`
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-touch-callout

```suggestion
  user-select: none;
  -webkit-touch-callout: none;
```